src/libs/ReportUtils.js

```diff
@@ -785,10 +786,11 @@ function getIconsForParticipants(participants, personalDetails) {
  * Given a report, return the associated workspace icon.
  *
  * @param {Object} report
+ * @param {Object} [policy]
  * @returns {Object}
  */
-function getWorkspaceIcon(report) {
-    const workspaceName = getPolicyName(report);
+function getWorkspaceIcon(report, policy = undefined) {
```

Comment:
I think `policy = undefined` has no meaning. We can safely remove the default value because default value is undefined in JS

Reply:
while its true, we found in other places that its more expressive. It conveys the meaning, that you don't have to set policy.
See similar cases here: https://github.com/Expensify/App/pull/23206/files#r1268753442

https://github.com/Expensify/App/blob/450573ebd01e710023940232da079928b41726a3/.github/libs/ActionUtils.js#L28